test/Microsoft.Framework.ConfigurationModel.Test/ConfigurationTest.cs

```diff
@@ -321,5 +322,16 @@ public void CanIterateAfterCastedToIEnumerable()
 
             Assert.Equal(3, srcCount);
         }
+
+        [Fact]
+        public void SetValueThrowsExceptionNoSourceRegistered()
+        {
+            var config = new Configuration();
+            var ex = Assert.Throws<InvalidOperationException>(() => config.Set("Title", "Welcome"));
+            Assert.Equal(
```

Comment:
Sure. Instead of adding the static error message, I will read error message from `Resources` and compare `ex.Message` to that.